packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -133,22 +135,31 @@ const AccountTransferForm = ({
     accounts_list.forEach((account, idx) => {
         const text = <AccountOption idx={idx} account={account} />;
         const value = account.value;
+        const currency = account.currency;
         (account.is_mt ? mt_accounts_from : accounts_from).push({
             text,
             value,
+            currency,
             nativepicker_text: `${account.text} (${account.currency} ${account.balance})`,
         });
         const is_selected_from = account.value === selected_from.value;
         // account from and to cannot be the same
         if (!is_selected_from) {
             const is_selected_from_mt = selected_from.is_mt && account.is_mt;
             const is_selected_from_crypto = selected_from.is_crypto && account.is_crypto;
-            // cannot transfer to MT account from MT
-            // cannot transfer to crypto account from crypto
-            const is_disabled = is_selected_from_mt || is_selected_from_crypto;
+            let is_disabled = false;
+            if (!is_deriv_crypto) {
```

Comment:
can we do `if (is_deriv_crypto)` instead, it's easier to read

Reply:
Done.